tests/Platforms/MySQL/ComparatorTest.php

```diff
@@ -9,17 +9,19 @@
 use Doctrine\DBAL\Platforms\MySQL\Comparator;
 use Doctrine\DBAL\Platforms\MySQL\DefaultTableOptions;
 use Doctrine\DBAL\Platforms\MySQLPlatform;
+use Doctrine\DBAL\Schema\ComparatorConfig;
 use Doctrine\DBAL\Tests\Schema\AbstractComparatorTestCase;
 
 class ComparatorTest extends AbstractComparatorTestCase
 {
-    protected function setUp(): void
+    protected function createComparator(ComparatorConfig $config): \Doctrine\DBAL\Schema\Comparator
```

Comment:
Please rely on a use statement for the return type declaration, just like you did for the parameter type declaration. We don't usually use FQCNs.

Reply:
Done in a6708ea1f20f4b5a6259313b13b1b242035e1957

Reply:
Now after changing the code I noticed that I could instead also just declare the `MySQL\Comparator` as the return type as it inherits from the `Schema\Comparator` anyways. Should I do that instead?

Reply:
Changed in 23f311977c3b854ed24c1f612d83462cf111ddcd

Reply:
Yes, good call!